src/components/Firestore/index.tsx

```diff
@@ -68,6 +67,11 @@ export const Firestore: React.FC<Props> = ({ config, projectId }) => {
     api.nukeDocuments();
   }
 
+  function handleNavigate(path: string) {
+    // TODO: needs a better impl
+    window.location.href = `/firestore/${path}`;
```

Comment:
Please render a `<Redirect>` or call `history.pushState` (with `useHistory()` I believe).

Reply:
Ah! wasn't sure the best way, thanks.

Reply:
Done.